lib/versioncheck.php

```diff
@@ -1,15 +1,15 @@
 <?php
 
 // Show warning if a PHP version below 7.2 is used,
-if (version_compare(PHP_VERSION, '7.2') === -1) {
+if (PHP_VERSION_ID < 70200) {
 	http_response_code(500);
 	echo 'This version of Nextcloud requires at least PHP 7.2<br/>';
 	echo 'You are currently running ' . PHP_VERSION . '. Please update your PHP version.';
 	exit(-1);
 }
 
 // Show warning if > PHP 7.4 is used as Nextcloud is not compatible with > PHP 7.4 for now
-if (version_compare(PHP_VERSION, '7.5.0') !== -1) {
+if (PHP_VERSION_ID >= 80000) {
```

Comment:
```suggestion
if (PHP_VERSION_ID > 70500) {
```

Feels cleaner because we limit to 7.4... I know 8 will be next but you never know what crazy stuff they come up with.

Reply:
'>=' of course :S